tests/cli/main.cpp

```diff
@@ -506,21 +501,11 @@ BOOST_FIXTURE_TEST_CASE( account_history_pagination, cli_fixture )
 {
    try
    {
-      INVOKE(upgrade_nathan_account);
-
-      // create a new account
-      graphene::wallet::brain_key_info bki = con.wallet_api_ptr->suggest_brain_key();
-      BOOST_CHECK(!bki.brain_priv_key.empty());
-      signed_transaction create_acct_tx = con.wallet_api_ptr->create_account_with_brain_key(
-         bki.brain_priv_key, "jmjatlanta", "nathan", "nathan", true
-      );
-      // save the private key for this new account in the wallet file
-      BOOST_CHECK(con.wallet_api_ptr->import_key("jmjatlanta", bki.wif_priv_key));
-      con.wallet_api_ptr->save_wallet_file(con.wallet_filename);
+      INVOKE(create_new_account);
 
       // attempt to give jmjatlanta some bitsahres
       BOOST_TEST_MESSAGE("Transferring bitshares from Nathan to jmjatlanta");
-      for(int i = 1; i <= 200; i++)
+      for(int i = 1; i <= 199; i++)
```

Comment:
Why this change? Something was broken?

Reply:
nothing was broken, I've changed this because we have +1 transaction inside INVOKE create_new_account, and I've decided to decrease here history by -1 transaction. Should I change another way ? I can change BOOST_CHECK history transfers to 202 as an example here
https://github.com/bitshares/bitshares-core/blob/18a78a5e614c941dbdafd3219b6476efdac9c13a/tests/cli/main.cpp#L518

what are your thoughts ? Thanks !

Reply:
>I've changed this because we have +1 transaction inside INVOKE create_new_account, and I've decided to decrease here history by -1 transaction. 

Makes sense. Thanks.